.travis.yml

```diff
@@ -4,18 +4,30 @@ language: python
 cache:
   directories:
   - "~/.cache/pip"
+
 python:
   - "2.7"
   - "3.4"
   - "3.5"
   - "3.6"
   - "pypy"
+
 install:
   - pip install -U pip setuptools>=18.5
   - pip install -r requirements-dev.txt
+
+matrix:
+  include:
+    - python: 2.7
+      env: MODE=lint
+    - python: 2.7
+      env: MODE=vendorverify
+    - python: 3.4
+      env: MODE=lint
```

Comment:
Turns out we weren't running lint in Travis. This fixes that as well as running vendorverify.

We lint in both 2.7 and 3.4 because flake8 does different things between Python 2 and 3.